go-controller/pkg/ovn/pods.go

```diff
@@ -183,6 +185,11 @@ func getRoutesGatewayIP(pod *kapi.Pod, gatewayIPnet *net.IPNet) ([]util.PodRoute
 			route.NextHop = gatewayIPnet.IP
 			routes = append(routes, route)
 		}
+		if namespaceExternalGW != "" {
```

Comment:
@rcarrillocruz I don't see any validation of the external gateway here. If that gateway is "asdfkjasdfa;lkdsfjasdf" then we'll basically blackhole traffic. Is that intended?

Reply:
so  what kind of validation? Would it suffice the annotation is an IP within the node range? or you just want an IPv4 validation?